pkg/pipeline/WorkflowService.go

```diff
@@ -112,6 +114,9 @@ type WorkflowRequest struct {
 	CiBuildDockerMtuValue      int                               `json:"ciBuildDockerMtuValue"`
 	IgnoreDockerCachePush      bool                              `json:"ignoreDockerCachePush"`
 	IgnoreDockerCachePull      bool                              `json:"ignoreDockerCachePull"`
+	CacheInvalidate            bool                              `json:"cacheInvalidate"`
+	IsPvcMounted               bool                              `json:"IsPvcMounted"`
+	PvcCachePath               string                            `json:"pvcCachePath"`
```

Comment:
why this flag is needed

Reply:
I didn't introduce the flag, but it's not being used anywhere so removed it